rkt/run.go

```diff
@@ -189,11 +189,10 @@ func runRun(cmd *cobra.Command, args []string) (exit int) {
 		return 1
 	}
 
-	if len(flagPodManifest) > 0 && (len(flagPorts) > 0 || rktApps.Count() > 0 || flagStoreOnly || flagNoStore ||
-		flagInheritEnv || !flagExplicitEnv.IsEmpty() || !flagEnvFromFile.IsEmpty() ||
-		(*appsVolume)(&rktApps).String() != "" || (*appMount)(&rktApps).String() != "" || (*appExec)(&rktApps).String() != "" ||
-		(*appUser)(&rktApps).String() != "" || (*appGroup)(&rktApps).String() != "" ||
-		(*appCapsRetain)(&rktApps).String() != "" || (*appCapsRemove)(&rktApps).String() != "") {
+	if len(flagPodManifest) > 0 && (rktApps.Count() > 0 ||
```

Comment:
@lucab It's not changing the semantics. I just removed unnecessary checks because they are taken cared by `parseApps()`.

```shell
$ sudo rkt run --pod-manifest=/dev/null coreos.com/etcd:v2.0.8 --environment=foo=bar
run: conflicting flags set with --pod-manifest (see --help)

$ sudo rkt run --pod-manifest=/dev/null --environment=foo=bar
Error: invalid argument "foo=bar" for --environment=foo=bar: --environment must follow an image
```

Did I miss anything?